src/View/Helper/FormHelper.php

```diff
@@ -217,11 +218,11 @@ class FormHelper extends Helper
     protected $_unlockedFields = [];
 
     /**
-     * Registry for input widgets.
+     * Locator for input widgets.
      *
-     * @var \Cake\View\Widget\WidgetRegistry
+     * @var \Cake\View\Widget\WidgetLocator
      */
-    protected $_registry;
+    protected $_locator;
```

Comment:
Is that change BC?

Reply:
It is highly unlikely to be relied on, but to be sure we can sure keep the former name here.
Note that we usually do not provide full BC on the internal protected methods (kind of private), especially in terms of low level infrastructure.
This would need a note in the migration guide then however.